.pre-commit-config.yaml

```diff
@@ -20,7 +20,7 @@ repos:
     rev: 19.10b0
     hooks:
     - id: black
-      language_version: python3.7
+      language_version: python3.8
```

Comment:
This should not change.

Reply:
I changed the version back to python3.7. Do we have to pin the version to python3.7 as the default python version has be specified as python3 on the top of the pre-commit-config.yaml file?

Reply:
cc @blefaudeux who uses pre-commit. I don't use pre-commit so not sure about this.

Reply:
we should use the same as what the CI is using, that's the only rule I know. CI seems to be using black directly, not specifying the language version, so I don't think that it changes much in practice (for this particular repo)